packages/salesforcedx-vscode-core/README.md

```diff
@@ -17,7 +17,29 @@ When a command finishes running (due to success, failure, or cancellation), a no
 ![Notification that source was successfully pushed to a scratch org](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/command_success_notification.png)
 
 To see the output of the commands that you run, select **View** > **Output**, and then select **Salesforce DX CLI** from the dropdown menu. Alternatively, click **Show** in the completion notification.  
-![Output view, showing the results of an Apex test run](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/output_view.png)
+![Output view, showing the results of an Apex test run](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/output_view.png)  
+
+These Salesforce CLI commands are available:
```

Comment:
I think this is missing:
force:source:status -> SFDX: View All Changes (Local and in Default Scratch Org)
force:source:status --remote -> SFDX: View Changes in Default Scratch Org
force:source:status --local -> SFDX: View Local Changes
force:apex:execute -> SFDX: Execute Anonymous Apex with Editor Contents
force:apex:execute -> SFDX: Execute Anonymous Apex with Currently Selected Text
force:data:soql:query -> SFDX: Execute SOQL Query with Currently Selected Text

The last two that say "Currently Selected Text" require the user to select some text in an opened editor for that command to show up.

cc @Sweetman in case we're missing any.

Reply:
Thanks, @DatGuyJonathan . I'm not sure why I didn't see the source:status commands in the command palette. I just added them to the list. I still don't see the `apex:execute` commands or the version of `data:soql:query` that uses the currently selected text. Are those going in this week?

Reply:
Yep, I see James' checkins in your branch for apex:execute and soql:query.

![execanoneditoropened](https://user-images.githubusercontent.com/7286437/31105987-e41a5614-a79c-11e7-881d-b3bb54311dc5.png)

![execanoneditortext](https://user-images.githubusercontent.com/7286437/31105991-e918c308-a79c-11e7-87ca-b5bf4acfb893.png)

Reply:
Thanks for checking, @DatGuyJonathan . I've added the missing commands to the list.